beluga-mqtt/src/lib.rs

```diff
@@ -95,6 +97,20 @@ impl<'a> MqttClientBuilder<'a> {
         self
     }
 
+    /// Sets the connect timeout.
+    ///
+    /// This timeout is used when connecting to the broker.
+    /// The value only has a resolution of seconds.
+    ///
+    /// Any value less than 1 second will be treated as 0 seconds,
+    /// which will cause the connection to fail instantly.
+    ///
+    /// Defaults to 5 seconds.
```

Comment:
that's peanuts, but you could link the actual default value `DEFAULT_CONNECT_TIMEOUT` here